vendors/default/style.scss

```diff
@@ -1 +1,2 @@
-$PRIMARY_COLOR: #2657FF;
\ No newline at end of file
+$PRIMARY_COLOR: #2657FF;
+$BREAKPOINT: 900px;
```

Comment:
Why not hardcode the breakpoint. Is there ever a situation we might want to change this?